evennia/utils/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.dispatch import Signal
```

Comment:
1. This module needs a doc string explaining how to use these signals (how to hook into them etc) without having to refer to Django documentation (there needs to be a wiki page in the future too). 
2. Maybe it would be a good idea to gather the default Django signals  (pre/post save etc) in here too, to have them all in one place?
3. If doing the above, I wonder if `evennia/server/signals.py` may not be a better location for this; these are sort of central things sent out by the server. (not sure about that one though)